app/io/pathfinder/controllers/PreFlight.java

```diff
@@ -0,0 +1,18 @@
+package io.pathfinder.controllers;
```

Comment:
It adds cors headers to all server responses, allowing us to make requests from a web app to a locally-running server. As mentioned above, this may be redundant as I had to adjust the implementation to also work with the scala controllers.